common/ast/src/parser/statement.rs

```diff
@@ -717,3 +821,20 @@ pub fn auth_type(i: Input) -> IResult<AuthType> {
         value(AuthType::JWT, rule! { JWT }),
     ))(i)
 }
+
+// parse: (k = v ...)* into a map
+pub fn options(i: Input) -> IResult<BTreeMap<String, String>> {
+    let ident_with_format = alt((
+        map(rule! { FORMAT }, |_| "FORMAT".to_string()),
+        map(ident, |ident| ident.to_string()),
```

Comment:
This may parse quoted ident as well.

Reply:
you can verify the ident using `and_then`: https://github.com/datafuselabs/databend/blob/7c7d55e1823e12a1104d4e0475f4c69a79c15f9e/common/ast/src/parser/expr.rs#L49